ada_2_fhir-r4/cio/2.0.0/sturen_voorstel_contra_indicatie/fhir_instance/cio-svci-tst-1.1-VCI1.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bundle xmlns="http://hl7.org/fhir">
+   <id value="cio-svci-tst-1.1-VCI1"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-ProposalContraIndication-Bundle"/>
+   </meta>
+   <type value="transaction"/>
+   <total value="6"/>
+   <link>
+      <relation value="self"/>
+      <url value="http://dummy.nictiz.nl/dummyquery"/>
+   </link>
+   <entry>
+      <fullUrl value="http://example.nictiz.nl/fhir/Flag/vci-pat06-vci-01"/>
+      <resource>
+         <Flag>
```

Comment:
ext-MedicationContraIndication.RelationAlert is missing inside Flag resource (but this is not required)